e2e/tests/node_reboot.robot

```diff
@@ -75,20 +91,25 @@ Power Off Node One By Once For More Than Pod Eviction Timeout While Workload Hea
     END
 
 Reboot All Worker Nodes While Workload Heavy Writing
```

Comment:
This test case failed with message:
```
AttributeError: 'NodeInstance' object has no attribute 'list_node_names_by_role'
```
Could you help check it? Thank you!
https://ci.longhorn.io/job/private/job/longhorn-e2e-test/375/robot/Tests/Node%20Reboot/Reboot%20All%20Worker%20Nodes%20While%20Workload%20Heavy%20Writing/

Reply:
Thanks, should be fixed now.